install_help.sh

```diff
@@ -117,6 +117,10 @@ BuildBE() {
   zfs set mountpoint=legacy $RPOOL/ROOT/omnios
   log "Cleaning up boot environment"
   beadm mount omnios /mnt
+  # generate UUID for BE
+  BEUUID=`LD_LIBRARY_PATH=/mnt/lib:/mnt/usr/lib /mnt/usr/bin/uuidgen`
+  zfs set org.opensolaris.libbe:uuid=$BEUUID $RPOOL/ROOT/omnios
+  zfs set org.opensolaris.libbe:policy=static $RPOOL/ROOT/omnios
   ALTROOT=/mnt
```

Comment:
I know this is not your code, but while looking at it ... why is ALTROOT not set after line 118 and then used instead of /mnt ?